app/api/utility.py

```diff
@@ -62,6 +67,11 @@
 
 IS_CONTROL_TERM = "ncit:C94342"
 
+BACKUP_VOCAB_DIR = (
+    Path(__file__).absolute().parents[2] / "vocab/backup_external"
+)
+VOCAB_FILE_PREFIX = {"cogatlas": "cogatlas_task"}
```

Comment:
similar to my comments on the path being assembled from this everywhere: I'd remove this bit here and either hardcode the path or store it on the state object as well. If you think this is going to be important soon, then you could also store the path as the value here, but I don't think having `cogatlas` -> `cogatlas_task` is helpful for the increase in reading complexity it brings